shakenfist/instance.py

```diff
@@ -385,17 +385,18 @@ def create(self, lock=None):
         with util.RecordedOperation('create domain', self):
             if not self.power_on():
                 attempts = 0
-                while not self.power_on() and attempts < 100:
+                while not self.power_on() and attempts < 5:
                     self.log.warning(
                         'Instance required an additional attempt to power on')
                     time.sleep(5)
                     attempts += 1
 
         if self.is_powered_on():
             self.log.info('Instance now powered on')
+            self.state = self.STATE_CREATED
         else:
             self.log.info('Instance failed to power on')
-        self.state = self.STATE_CREATED
+            self.enqueue_delete_due_error('Instance failed to power on')
```

Comment:
Should this be using the STATE_CREATING_ERROR? Or somehow indicating failure.

Reply:
That's done in enqueue_delete_due_error.